Projects/ChangeTier/Sources/Navigation/ChangeTierNavigation.swift

```diff
@@ -10,7 +10,7 @@ public class ChangeTierNavigationViewModel: ObservableObject {
     @Published public var isCompareTiersPresented = false
     @Published public var isInsurableLimitPresented: InsurableLimits?
     @Published public var document: Document?
-
+    let useOwnNavigation: Bool
     let router: Router
 
     //make sure to set it!!!
```

Comment:
can we remove this comment?

Reply:
This is like info, can we keep it?

Reply:
Ah yes but then I think we should rephrase it to look more professional. Maybe "Note: important to set this." or something like that